distill/core/log.py

```diff
@@ -57,4 +58,37 @@ def to_json(self) -> str:
 
     def to_dict(self) -> JsonDict:
         return self.data.model_dump(by_alias=True)
-
+    
+    def __lt__(self, other):
+        if isinstance(other, Log):
+            return self.id < other.id
+        if isinstance(other, datetime):
+            return self.id.get_datetime() < other.now()
+    
+    def __le__(self, other):
+        if isinstance(other, Log):
+            return self.id <= other.id
+        if isinstance(other, datetime):
+            return self.id.get_datetime() <= other.now()
```

Comment:
Same as above for every method

Reply:
Response is the same as my comment above.